src/js/jsx/search/SearchBar.jsx

```diff
@@ -180,11 +180,18 @@ define(function (require, exports, module) {
 
                     var title = option.title.toLowerCase(),
                         category = option.category || [];
+ 
+                    if (option.id.indexOf("FILTER") === 0) {
```

Comment:
I know this isn't new, but we should figure out a better way to distinguish between filter and regular options than with a string comparison. What about if filter options just have a true `filter` property?